### PR TITLE
Naive approximation to multiple-point lines (not full-bezier support)

### PR DESCRIPTION
**Pull request by ralequi**

### CODE REVIEW

**Comment by ralequi** [comment]
Compilation issues that you should check:

* Python 3.4 is failing to dowload on travis. You probably want to remove it from .travis-ci.
* Python 3.6 require to dowload "black" package. It is failing to do it. You probably want to remove that line from .travis-ci

**Comment by ralequi** [comment]
Added some minor fixes/workarounds for some dot versions

**Comment by hbmartin** [comment]
@ralequi this looks really cool, thank you! I’ll look at to get this merged in the next couple of days. 

In the meantime, could you provide tests or examples?
(Sorry if I missed something, writing from my phone)

**Comment by ralequi** [comment]
> @ralequi this looks really cool, thank you! I’ll look at to get this merged in the next couple of days.
> 
> In the meantime, could you provide tests or examples?
> (Sorry if I missed something, writing from my phone)

Hi,

That was my main idea when i see the codecov messages..., however I looked that there was only one very simple test.
I can add an example and a few lines so my new lines run under the tests.
In a few minutes I would commit it

File: test/test_graphs.py [hunk 1]
```diff
@@ -61,3 +62,18 @@ def test_hello():
...
     check_edge_dir(edge, dx=0, dy=1)
+
+
+def test_polylines():
+    file = "./undirected/polylines.gv.txt"
+    xml = graphviz2drawio.convert(file)
+    print(xml)
+
+    root = ET.fromstring(xml)
+    elements = check_xml_top(root)
```
Comment (hbmartin) [inline test/test_graphs.py]: Looks like a much better way to check 👍 
But why remove the testing on the other `undirected`?

**Comment by hbmartin** [comment]
Merged and cut a new release with this! Feel free to add yourself to the Authors section on the Readme 😄

File: test/test_graphs.py (hunk 1 above)
Comment (ralequi) [reply test/test_graphs.py]: what do you mean with remove other testing?
I would love to execute every test (at least to check that every .dot generates its own xml) but many of them do not work. Some, even require external images for example

**Pull request merged**